ROADMAP.md

```diff
@@ -0,0 +1,123 @@
+xk6-browser roadmap
```

Comment:
It might be good to add a graphical timeline to better visualize this. WDYT?

Reply:
We could use the help from our design department for this. :) @sniku Do you think this is a good idea?

Reply:
It could be a nice addition, but let's consider this after the PR is merged :smile: